recipes/grain/build.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+
+set -xe
+export PYTHON_VERSION=$(${PYTHON} -c "import sys; print(f'{sys.version_info.major}.{sys.version_info.minor}')")
+export PYTHON_MAJOR_VERSION=$(echo $PYTHON_VERSION | cut -d. -f1)
+export PYTHON_MINOR_VERSION=$(echo $PYTHON_VERSION | cut -d. -f2)
+export BAZEL_VERSION="7.2.1"
+export OUTPUT_DIR="$(pwd)"
+export SOURCE_DIR="."
+export RUN_TESTS="true"
+. "./grain/oss/runner_common.sh"
+
+#build_and_test_grain
+setup_env_vars_py "$PYTHON_MAJOR_VERSION" "$PYTHON_MINOR_VERSION"
+
+function write_to_bazelrc() {
+  echo "$1" >> .bazelrc
+}
+
+write_to_bazelrc "build --incompatible_default_to_explicit_init_py"
+write_to_bazelrc "build --enable_platform_specific_config"
+write_to_bazelrc "build:macos --linkopt=-Wl,-undefined,dynamic_lookup"
+write_to_bazelrc "build:macos --host_linkopt=-Wl,-undefined,dynamic_lookup"
+write_to_bazelrc "build --@rules_python//python/config_settings:python_version=${PYTHON_VERSION}"
+write_to_bazelrc "build --cxxopt=-Wno-deprecated-declarations --host_cxxopt=-Wno-deprecated-declarations"
+write_to_bazelrc "build --cxxopt=-Wno-parentheses --host_cxxopt=-Wno-parentheses"
+write_to_bazelrc "build --cxxopt=-Wno-sign-compare --host_cxxopt=-Wno-sign-compare"
+
+write_to_bazelrc "test --@rules_python//python/config_settings:python_version=${PYTHON_VERSION}"
+write_to_bazelrc "test --action_env PYTHON_VERSION=${PYTHON_VERSION}"
+write_to_bazelrc "test --test_timeout=300"
+write_to_bazelrc "common --check_direct_dependencies=error"
+
+bazel clean
+bazel build ... --action_env PYTHON_BIN_PATH="${PYTHON_BIN}" --action_env MACOSX_DEPLOYMENT_TARGET='11.0'
```

Comment:
```suggestion
bazel build ... --action_env PYTHON_BIN_PATH="${PYTHON_BIN}" --action_env MACOSX_DEPLOYMENT_TARGET=${MACOSX_DEPLOYMENT_TARGET}
```

This is usually set by conda-build's orchestration. See https://github.com/mtsokol/staged-recipes/blob/9c41fee30a16afda74c4fbf61ea4d19b7d636eb7/.ci_support/osx64.yaml#L9-L10.